startup/src/main/java/tech/okcredit/startup_instrumentation/AppStartUpTrace.kt

```diff
@@ -0,0 +1,71 @@
+package tech.okcredit.startup_instrumentation
+
+import android.app.Application
+import android.content.Context
+import android.os.Build
+import android.os.SystemClock
+import org.jetbrains.annotations.NonNls
+import tech.okcredit.startup_instrumentation.internals.AppStartMeasureLifeCycleCallBacks
+import tech.okcredit.startup_instrumentation.internals.AppStartUpMeasurementUtils
+import java.util.concurrent.Executors
+
+object AppStartUpTrace {
```

Comment:
This class has a state. Should we make methods instance method instead of static?

Reply:
I didn't get this

Reply:
Should we make it a class instead of object since it's stateless?

Reply:
it's indented. we needed to access this class from multiple places, like content provider, App.class, etc and keep values in memory. we wanted to have only one instance throughout the App. it added 5x8(5 long variables) in memory. Any better solution here?